distribution/docker/src/docker/Dockerfile.cloud-ess

```diff
@@ -10,3 +10,4 @@ RUN chmod 0444 /opt/plugins/archive/*
 FROM ${base_image}
 
 COPY --from=builder /opt/plugins /opt/plugins
+ENV ELASTICSEARCH_PLUGIN_ARCHIVE_DIR /opt/plugins/archive
```

Comment:
Should we use the typical `ES` prefix instead of `ELASTICSEARCH`?

Reply:
It's purely an implementation detail, but I've changed it to `ES_` anyway.